client/main.lua

```diff
@@ -1,4 +1,3 @@
-local QBCore = exports['qbx-core']:GetCoreObject()
 local PlayerJob = {}
```

Comment:
This can be removed with all the other PlayerJob instances being changed to QBX.PlayerData.job